.nvmrc

```diff
@@ -1 +1 @@
-18
+20.11.1
```

Comment:
@clmedders @nick-mon1 we should check in on these versions in POAM.

Reply:
will add in poam doc to capture this!